src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -13,13 +13,19 @@ internal static partial class GlobalizationMode
         private static partial class Settings
         {
             internal static bool Invariant { get; } = AppContextConfigHelper.GetBooleanConfig("System.Globalization.Invariant", "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT");
+#if TARGET_OSX || TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            internal static bool Hybrid { get; } = AppContextConfigHelper.GetBooleanConfig("System.Globalization.Hybrid", "DOTNET_SYSTEM_GLOBALIZATION_HYBRID");
```

Comment:
If `HybridGlobalization` is not going to be a feature switch, should this be using `AppContextConfigHelper.GetBooleanConfig`? Will the switch `System.Globalization.Hybrid` ever be set? I think `System.Globalization.Invariant` and `System.Globalization.PredefinedCulturesOnly` switches are probably set in the `TrimmingTests` https://github.com/dotnet/runtime/blob/9b38f2ab7440b551e294ff43eea20ef8866c931a/src/libraries/System.Runtime/tests/TrimmingTests/System.Runtime.TrimmingTests.proj#L16-L19.

Reply:
Once more functionalities will be implemented this will be used and we can have tests in TrimmingTests for hybrid mode.